integration/error-boundary-test.ts

```diff
@@ -875,3 +875,329 @@ test.describe("loaderData in ErrorBoundary", () => {
     });
   }
 });
+
+test.describe("Default ErrorBoundary", () => {
```

Comment:
Document request + SPA transition tests for:

* no root boundary
* root boundary
* root boundary that throws